src/ppom-controller.ts

```diff
@@ -257,34 +285,15 @@ export class PPOMController extends BaseControllerV2<
     this.#startDataRefreshTask();
   }
 
-  /**
-   * Set the interval at which the ppom version info will be fetched.
-   * Fetching will only occur on the next call to test/bypass.
-   * For immediate update to the ppom lists, call updatePPOM directly.
-   *
-   * @param interval - The new interval in ms.
-   */
-  setRefreshInterval(interval: number) {
-    this.update((draftState) => {
-      draftState.refreshInterval = interval;
-    });
-    this.#startDataRefreshTask(interval);
-  }
-
-  /**
-   * Clears the periodic job to refresh file data.
-   */
-  clearRefreshInterval() {
-    clearInterval(this.#refreshDataInterval);
-  }
-
   /**
    * Update the PPOM.
    * This function will acquire mutex lock and invoke internal method #updatePPOM.
+   *
+   * @param updateForAllChains - True is update if required to be done for all chains in cache.
    */
-  async updatePPOM() {
+  async updatePPOM(updateForAllChains = true) {
```

Comment:
As this is a public method, would it be cleaner to put this boolean in an options object to make it more explicit when called?